packages/social-previews/src/nextdoor-preview/previews.tsx

```diff
@@ -0,0 +1,53 @@
+import { ExternalLink } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import SectionHeading from '../shared/section-heading';
+import { NextdoorLinkPreview } from './link-preview';
+import { NextdoorPostPreview } from './post-preview';
+import type { NextdoorPreviewsProps } from './types';
+
+export const NextdoorPreviews: React.FC< NextdoorPreviewsProps > = ( {
+	headingLevel,
+	hideLinkPreview,
+	hidePostPreview,
+	...props
+} ) => {
+	return (
+		<div className="social-preview nextdoor-preview">
+			{ ! hidePostPreview && (
+				<section className="social-preview__section nextdoor-preview__section">
+					<SectionHeading level={ headingLevel }>
+						{
+							// translators: refers to a social post on Nextdoor
+							__( 'Your post', 'social-previews' )
+						}
+					</SectionHeading>
+					<p className="social-preview__section-desc">
+						{ __( 'This is what your social post will look like on Nextdoor:', 'social-previews' ) }
+					</p>
+					<NextdoorPostPreview { ...props } />
+				</section>
+			) }
+			{ ! hideLinkPreview && (
+				<section className="social-preview__section nextdoor-preview__section">
+					<SectionHeading level={ headingLevel }>
+						{
+							// translators: refers to a link to a Nextdoor post
+							__( 'Link preview', 'social-previews' )
+						}
+					</SectionHeading>
+					<p className="social-preview__section-desc">
+						{ __(
+							'This is what it will look like when someone shares the link to your WordPress post on Nextdoor.',
+							'social-previews'
+						) }
+						&nbsp;
+						<ExternalLink href="https://jetpack.com/support/jetpack-social-image-generator">
+							{ __( 'Learn more about links', 'social-previews' ) }
+						</ExternalLink>
+					</p>
+					<NextdoorLinkPreview { ...props } name="" profileImage="" />
```

Comment:
I'm not sure if this problem is isolated to Nextdoor, but we're using the selected media for the link preview when a featured image is set on the post. I'm not sure that's correct, as I think the featured image will be used as the Open Graph image, unless Social Image Generator is enabled.

Reply:
IIRC, I asked this question before probably in a team meeting whether the custom media is used as OG image and the answer was "Yes".

Reply:
Also, if it's a problem, then it's with all the reviews then and needs a fix in Jetpack, not here.

Reply:
Sounds like it is working the right way! p1698924866197229/1698825180.539239-slack-C02JJ910CNL